doc/manual/overview.tex

```diff
@@ -0,0 +1,212 @@
+\tikzstyle{single} = [draw = brand, minimum width = 2 cm]
+\tikzstyle{splits} = [draw = brand, rectangle split, solid]
+\tikzstyle{linelabel} = [anchor = west, font=\sc]
+\tikzstyle{colorbox} = [rectangle, very thin, draw=gray, sharp corners, minimum height = 0.4 cm, minimum width = 1.8 cm]
+
+% color coding
+\tikzstyle{line} = [thick]
+\tikzstyle{network} = [line, draw = magenta]
+\tikzstyle{git} = [line, draw = violet]
+\tikzstyle{rtio} = [line, draw = brand-light]
+\tikzstyle{drtio} = [line, draw = blue]
+\tikzstyle{slow} = [line, draw = dingle]
+\tikzstyle{jtag} = [line, draw = gray]
+\tikzstyle{direct} = [line, draw = primary, thin]
+
+\tikzstyle{master} = [draw = purple]
+
+\tikzstyle{host} = [row sep = 0.5 cm, column sep = 0.7 cm]
+\tikzstyle{pc} = [matrices, thin, inner sep = 0.2cm, draw = gray, dashed]
+\tikzstyle{peripheral} = [matrices, draw = secondary]
+\tikzstyle{core} = [matrices, draw = purple]
+
+% PERIPHERALS (SLOW)
+
+\matrix[peripheral](peri-slow) at (0, 15.75)
+{
+  \node[label] (label){Peripherals (slow)};\\
+  \node [splits, rectangle split parts = 4](slow-list){
+    \nodepart{one} Optical translation stages
+    \nodepart{two} Optical rotation stages
+    \nodepart{three} Temperature controllers
+    \nodepart{four} etc...
+  };\\
+};
+
+% OTHER MACHINES
+
+\matrix[pc](comtools) at (6.38, 15.75)
+{
+    \node [splits, rectangle split parts = 3](remote-ctlrs){
+    \nodepart{one} Controller
+    \nodepart{two} Controller
+    \nodepart{three} etc...
+  }; \\
+  \node[single, minimum height = 0.5cm, font=\tt](remote-ctlmgr){
+    artiq\_ctlmgr\textsuperscript{2}
+  }; \\
+};
+
+\node[anchor = north west] (label) at (11, 16) {Client machines, NB: see footnote 1.};
+
+\matrix[pc](clients) at (10.75, 15)
+{
+   \node[single, draw=rtm, solid](remote-repo) {Cloned repository};
+   \&
+    \node [splits, master, rectangle split parts = 2, minimum width = 3 cm](remote-client){
+    \nodepart[font=\tt]{one} artiq\_client
+    \nodepart[font=\tt]{two} artiq\_dashboard
+  };
+  \\
+};
+
+% HOST MACHINE
+
+\matrix[host](host-machine) at (0, 11)
+{
+      \node[font=\tt, single](browser) {artiq\_browser};
+      \&
+      \node[single, master](results){Results};
+      \&
+      \node[splits, rectangle split parts = 2, master](master) {
+        \nodepart[font=\bf]{one} ARTIQ master
+        \nodepart{two} Datasets
+      };
+      \&
+      \node [single, master](repo){Repository};
+      \&
+      \node[font=\tt, single, master](ctlmgr) {artiq\_ctlmgr};
+      \\
+
+      \node[font=\tt, single](flash) {artiq\_flash};
+      \&
+      \node[font=\tt, single](run) {artiq\_run};
+      \&
+      \node[single, master](ddb) {Device database};
+      \&
+      \node[font=\tt, single](coremgmt) {artiq\_coremgmt\textsuperscript{3}};
+      \&
+      \node[single, master](builtin) {Built-in controllers\textsuperscript{4}};
+       \\
+};
+
+% BORDER LINE
+\node[linelabel] (label1) at (-0.25, 6.3) {pcs and remote devices};
```

Comment:
is it possible to make it appear as `PCs` instead of `PCS`? Latter may bring some confusion

Reply:
Kind of: 
![image](https://github.com/user-attachments/assets/1fcdc10b-9a23-452a-a010-100657e83df3)
as opposed to 
![image](https://github.com/user-attachments/assets/38d0bcb6-d69e-46ed-bd29-8c860d1531c3)

Not using smallcaps makes the font noticeably unbalanced. Of course it can also be: 

![image](https://github.com/user-attachments/assets/a6fa0e79-ebc8-423c-9ca8-a6707b760282)

but I must say I prefer the smallcaps.